boards/FSM/Src/App/states/App_AirClosedState.c

```diff
@@ -18,13 +18,49 @@ static void AirClosedStateRunOnTick(struct StateMachine *const state_machine)
         App_FsmWorld_GetPrimaryFlowMeter(world);
     struct FlowMeter *secondary_flow_meter =
         App_FsmWorld_GetSecondaryFlowMeter(world);
+    struct WheelSpeedSensor *left_wheel_speed_sensor =
+        App_FsmWorld_GetLeftWheelSpeedSensor(world);
+    struct WheelSpeedSensor *right_wheel_speed_sensor =
+        App_FsmWorld_GetRightWheelSpeedSensor(world);
 
     App_FlowMeter_Tick(primary_flow_meter);
     App_FlowMeter_Tick(secondary_flow_meter);
+
     App_SetPeriodicCanSignals_PrimaryFlowMeter(
         can_tx_interface, primary_flow_meter);
     App_SetPeriodicCanSignals_SecondaryFlowMeter(
         can_tx_interface, secondary_flow_meter);
+
+    const float left_wheel_speed =
```

Comment:
Actually what I would do here is to have one `SetCanPeriodicSignals` that reports the speed and fault, and use it in each state machine
```
void App_SetPeriodicCanSignal_Wheel(
struct Wheel* wheel, 
struct FsmCanTxInterface can_tx,
void (*wheel_speed_can_signal_setter(struct FsmCanTxInterface*, float),
void (*out_of_range_can_signal_setter)(struct FsmCanTxInterface*, bool),

{ 
  const float wheel_speed = get_wheel_speed(wheel);

  wheel_speed_can_signal_setter(can_tx, wheel_speed);

  if (wheel_speed > get_max_threshold(wheel)) 
  {
    out_of_range_can_signal_setter(can_tx, true);
  }
  else
  {
    out_of_range_can_signal_setter(can_tx, false);
  }
}

Reply:
I'd modify this with the suggestion I mentioned here: https://github.com/UBCFormulaElectric/Consolidated-Firmware/pull/582/files#r418246902